src/controllers/blocks/BlocksController.ts

```diff
@@ -18,6 +18,7 @@ import type { ApiPromise } from '@polkadot/api';
 import { isHex } from '@polkadot/util';
 import { RequestHandler } from 'express';
 import { BadRequest } from 'http-errors';
+import client from 'prom-client';
```

Comment:
If we are using `client` here just as a type - lets do `import type client from ''prom-client`